invenio_mail/config.py

```diff
@@ -19,3 +19,12 @@
 
 MAIL_MAX_ATTACHMENT_SIZE = 1000000
 """Max size of inline attachments, in bytes."""
+
+MAIL_MAX_RETRIES = 2
+"""How often will we repeat if a problem occurred."""
+
+MAIL_RETRY_COUNTDOWN = 180
```

Comment:
Shall we go for the classic exponential waiting instead a fixed amount of time?

Reply:
That was my first thought, too :-) Then I thought about making the decision configurable but didn't find a good solution and went instead for the default value. But you're right, an exponential waiting is better here.

Reply:
Replaced the static countdown with an exponential backoff with random jitter (to prevent thundering herd problem).

Reply:
I think that this config is not needed anymore?

Reply:
Ooops, forgot to remove!